spec/index.bs

```diff
@@ -1255,6 +1255,34 @@ For more information on issuing CORS requests for image and video elements, cons
 - [[html#the-img-element]] <{img}>
 - [[html#media-elements]] {{HTMLMediaElement}}
 
+## Color Spaces ## {#color-spaces}
+
+WebGPU does not provide color management. All values within WebGPU (such as texture elements)
+are raw numeric values, not color-managed color values.
+
+WebGPU *does* interface with color-managed outputs (via {{GPUCanvasContext}}) and inputs
+(via {{GPUQueue/copyExternalImageToTexture()}}).
+Color conversion must be performed between the WebGPU numeric values and the external color values.
+These interface points locally define a color space, in which the WebGPU numeric values are to be
+interpreted, from options defined in [[css-color-4#predefined]].
+
+<script type=idl>
+enum GPUPredefinedColorSpace {
+    "srgb",
+};
+</script>
+
+Issue: Replace this with the `PredefinedColorSpaceEnum` from the
+[canvas color space proposal](https://github.com/WICG/canvas-color-space/blob/main/CanvasColorSpaceProposal.md).
+
+Issue: The most urgent additional option to add here is "srgb-linear", needed to decode
+nonlinearly encoded data into linearly-encoded textures (e.g. float16 textures).
```

Comment:
That's not why srgb-linear is desired. Float16 really doesn't care about the output transfer function, since precision is good enough any-which-way. There's no inability to decode here.

Just strike this issue, since we have more to discuss here.

Reply:
I don't mean to say it's required for all float textures, but that it's required for a user to create a linearly encoded float format texture. It's equally applicable to linearly encoded unorm textures except that that's a rarer usecase (can use unorm-srgb instead).